indico/modules/events/templates/display/conference/base.html

```diff
@@ -11,14 +11,14 @@
                 <div class="confTitle">
                     <h1>
                         <a href="{{ event.url }}">
-                            <span class="conference-title-link" style="{{ conf_layout_params.text_color_css }}">
+                            <div class="conference-title-link" style="{{ conf_layout_params.text_color_css }}">
```

Comment:
This change breaks a few events that use `span.conference-title-link` in their custom CSS. Since it's just a bit ugly from a semantic point of view, I'd rather revert this to `span`...